### DISCUSSION TITLE
【个人猜测】米哈游可能又要吸取龙脉了

### DISCUSSION THREAD

**Discussion by match-fire** [share121/inter-knot#1407] [其他]
最近，我听内鬼说，ZZZ 2.0版本要出一个有中华龙要素的角色
到时候可能会跟隔壁饮月一样
像是在南京，北京等这样的大城市宣发
但不知大家察觉到了没有
这些城市里不乏有一些文明古都
那古都里可是有龙脉之气
再加上原神里的那维莱特（水龙）钟离（岩龙）崩铁里的饮月（虚数龙）
这不妥妥的四龙降世吗？？
各类传说都表明：四龙汇聚之时，则天下福分竭尽之际
米哈游莫不是又要褒姒，想要再吸收中华龙脉？？
你细细品一下

所以XDM，赶紧提桶跑路吧，别再玩米家的游戏了
说不定哪天褒姒你的老婆就都没了（悲

**Comment by lllliufeng** [comment]
我勒个龙脉2.0啊

**Comment by gaojiaobei** [comment]
我勒个龙脉啊

**Comment by hzjwork** [comment]
乆乆乆，mhy真是太可怕了

**Comment by WORLD-executeME** [comment]
赛博龙脉2.0，启动！

**Comment by Yishinaisy** [comment]
surprising

**Comment by lc367** [comment]
孩子们快跑别被骗了

**Comment by kallen1314** [comment]
其实我不是很爱看这些，感觉节奏一个接一个

**Comment by GanlinSama** [comment]
梗很烂，别引战😅

**Comment by qwq1949** [comment]
我直接抽爆Rick🤩

**Comment by FurinaShark** [comment]
打过来我第一个跑

**Comment by Nahida1205** [comment]
个人建议，拉踩的行为不应出现在绳网，社区氛围应是对游戏内容的和谐讨论，而非聚焦于对其它游戏的贬低与嘲讽，或许你只是想单纯的玩梗，但它在这个时间出现在这里仍是不恰当的，这里是最后的“一心净土”，而非另一处战场，希望您能够规范自身的发言，主动维护这片和谐的环境，谢谢！

**Reply by match-fire** [reply]
?

**Reply by qwq1949** [reply]
点一下链接试试嘛~（）

**Reply by Nahida1205** [reply]
嘶，没注意这连接，我的问题

**Comment by su2569** [comment]
看到BV1G就开始笑了

**Comment by 42417** [comment]
个人感觉 11 楼说的有道理，游戏没有那么火本身也不是坏事，至少圈子干净一点点，社区安宁还得靠大家维护

**Comment by angelicaNdahurica** [comment]
评论有附和也有抵制，这很绳网😎
